docs/docs/tascomi-ingestion.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Tascomi data ingestion
+description: "Description of the ingestion and refinement pipeline for Tascomi planning data"
+layout: playbook_js
+tags: [playbook]
+---
+
+This section describes how Tascomi Planning data gets ingested and refined in the data platform. The process relies on [Tascomi API](https://hackney-planning.tascomi.com/rest/v1/documentation.html?public_key=dd95bcd473f46a4325a4021d54500c7d#available-resources) and is composed of the following steps:
+
+- An initial full ingestion from Tascomi API (only once, happened in October 2021)
+- A daily call to the Tascomi API to get latest updated records (increment)
+- Parsing of the json data increment returned by the API
+- Refinement of the parsed data to recast all columns to the right data type
+- Creation of a full snapshot by applying the daily increment to the previous snapshot
+
+## Details of individual steps
+### 0 - Initial full ingestion
+This [process](https://github.com/LBHackney-IT/Data-Platform/blob/main/scripts/jobs/planning/tascomi_api_ingestion.py) queries one API endpoint (e.g. the applications endpoint) and writes the data into a table of the same name.
+
+This process wrote into the raw zone, in the 'api_response' bucket. The data is partitioned by import_date.
```

Comment:
i would say, "raw zone buket, with the prefix 'api_response'

Reply:
And use `import_date` instead of import_date,

Reply:
done